build.sbt

```diff
@@ -411,11 +411,11 @@ lazy val compilerOptionsExporter = Project("compilerOptionsExporter", file(".")
   .settings(disablePublishing)
   .settings(
     libraryDependencies ++= Seq(
-      "com.fasterxml.jackson.core" % "jackson-core" % "2.13.0",
-      "com.fasterxml.jackson.core" % "jackson-annotations" % "2.13.0",
-      "com.fasterxml.jackson.core" % "jackson-databind" % "2.9.10.8",
-      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.9.10",
-      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.0"
+      "com.fasterxml.jackson.core" % "jackson-core" % "2.13.1",
+      "com.fasterxml.jackson.core" % "jackson-annotations" % "2.13.1",
+      "com.fasterxml.jackson.core" % "jackson-databind" % "2.13.1",
+      "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.13.1",
+      "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.0",
```

Comment:
2.13.0 is actually the newest available version of jackson-module-scala. But I should have added a comment saying so, since it looks so much like a mistake. I'll add the comment.

I don't know enough about Jackson to know if the version mismatch is somehow a problem, but it's only used in `compilerOptionsExporter`, so it's low risk. I did try `compilerOptionsExporter/run` and it does still seem to work.

I also added the variable you suggested, and I hope the Steward can figure it out!

Reply:
gah, if jackson-module-scala 2.13.1 doesn't exist, why did the Steward open #9872?

looking into it...

Reply:
Ah, it must be so new that it doesn't show up in search.maven.org yet. But it does exist. (It's normal for search.maven.org to lag a bit.)

Okay, I'll revise this again.